server/user_cloud.go

```diff
@@ -4,14 +4,13 @@
 package main
 
 import (
+	"github.com/mattermost/mattermost-server/model"
```

Comment:
nit: I think we have the convention of putting all MM imports at the bottom

Reply:
Thanks for pointing that out -- I think that was automatic. I've added go imports to my global file watchers now.